package.json

```diff
@@ -123,6 +123,7 @@
     "serve-notify": "export NOTIFY=true && npm run serve",
     "scaffold": "./node_modules/.bin/gulp ensureConfig && ./node_modules/.bin/gulp ensureDevProxy",
     "start": "NODE_ENV=development webpack-dev-server --config ./webpack/webpack.dev.babel.js --hot --progress --colors --content-base ./build --host 0.0.0.0 --port $npm_package_config_port",
+    "start-reactjs-components-local": "REACTJS_COMPONENTS_LOCAL=true npm start",
```

Comment:
I would prefer we add this to the `CONTRIBUTING.md` on how to develop with reactjs-components, instead of creating a script here.